dds/DCPS/transport/framework/TransportSendBuffer.cpp

```diff
@@ -370,6 +370,12 @@ SingleSendBuffer::check_capacity_i(BufferVec& removed)
   }
 }
 
+bool
+SingleSendBuffer::has_frags(const SequenceNumber& seq) const
+{
+  return fragments_.find(seq) != fragments_.end();
```

Comment:
tomAYto / tomAHto

I think I prefer the aesthetic of using find rather than count, since I don't really care about the number. Looking at the implementation, count just wraps what I've written with an extra layer that returns `1` or `0` depending on the comparison. For C++14 or greater, there's also the potential confusion of accidentally triggering the templated version of count which allows for direct comparisons of non-key types against the keys in the map... something to avoid when we actually have the correct key type.

Reply:
Looking at cppreference, it looks like find also gets the templated non-key version in C++14, so I guess it really boils down to whether the brevity of `count` is worth the wrapping.

Reply:
Not a big deal at all (I didn't make it "requested changes") -- in general, without knowing anything about how it's implemented, prefer using a simpler function.  With map (as opposed to multimap) count semantically returns a bool (0 or 1) but the type is int to make it compatible with multimap.